tests/integration/docusaurus/connecting_to_your_data/cloud/azure/spark/configured_yaml_example.py

```diff
@@ -0,0 +1,86 @@
+import os
+from typing import List
+
+from ruamel import yaml
+
+import great_expectations as ge
+from great_expectations.core.batch import Batch, BatchRequest
+
+credential = os.getenv("AZURE_ACCESS_KEY", "")
+
+context = ge.get_context()
+
+datasource_yaml = f"""
+name: my_azure_datasource
+class_name: Datasource
+execution_engine:
+    class_name: SparkDFExecutionEngine
+    azure_options:
+        credential: <YOUR_CREDENTIAL>
+data_connectors:
+    configured_data_connector_name:
+        class_name: ConfiguredAssetAzureDataConnector
+        azure_options:
+            credential: <YOUR_CREDENTIAL>
+        container: <YOUR_AZURE_CONTAINER_HERE>
+        name_starts_with: <CONTAINER_PATH_TO_DATA>
```

Comment:
Same note as above. We do this here so perhaps let's make it consistent throughout?

Reply:
@cdkini This is YAML -- I believe we treat them differently.

Reply:
I don't believe that is the case with S3 and GCS actually. I could be wrong but I think the less specificity the better here (just to help the future doc writer)